docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -491,6 +491,26 @@ Type: Double
 
 Default: 0.9
 
+## max_bytes_ratio_before_external_group_by_for_server
```

Comment:
>This is a dangerous setting

Why do you find it dangerous? It can be enabled only by admins

> I'm not sure it's necessary and probably shouldn't be enabled by default
> It actually enables degradation of all requests on the server.

It is not enabled yet anyway, the main idea to enable it by default (later), is to avoid MEMORY_LIMIT_EXCEEDED errors, but I'm not insist here